docs_src/V3TopLevelMigration.md

```diff
@@ -84,6 +85,22 @@ If the additional service(s) in your custom compose file are EdgeX released devi
 
 The latest V3 Compose Builder can be found here: [Compose Builder Readme](https://github.com/edgexfoundry/edgex-compose/tree/{{version}}/compose-builder/README.md) 
 
+## Command-Line Options and Environment Variables
```

Comment:
do you want the name to be consistent with line 10 above?

Reply:
Just pushed changes that simplified section naming. :-)